tests/test_engine_initialization.py

```
@@ -756,6 +756,90 @@ def test_preloaded_scoped_conversation_manager_is_adopted_within_run_state() ->
     assert adopted is preloaded
 
 
+def test_scoped_conversation_manager_reports_scoped_context_window_usage() -> None:
+    from penguin.system.context_window import ContextWindowManager
+    from penguin.system.state import MessageCategory
+
+    scoped_window = ContextWindowManager(token_counter=lambda content: 0)
```

Ruff (0.15.13)

[warning] Unused lambda argument: `content`

(ARG005)